README.md

```
@@ -1,5 +1,7 @@
 # Lightspeed Core Stack (LCS)
 
+## About The Project
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)